packages/client/src/util/rpc.ts

```diff
@@ -63,51 +86,23 @@ export function createRPCServer(
   manager: RPCManager,
   opts: CreateRPCServerOpts
 ): CreateRPCServerReturn {
-  const { methodConfig, rpcDebug, logger } = opts
+  const { methodConfig, rpcDebug, rpcDebugVerbose, logger } = opts
   const onRequest = (request: any) => {
-    let msg = ''
-    if (rpcDebug && rpcDebug !== '') {
-      let show = false
-      if (rpcDebug === '' || rpcDebug === 'all') {
-        show = true
-      } else {
-        const filters = rpcDebug.split(',')
-        for (const filter of filters) {
-          if (request.method.includes(filter) === true) {
-            show = true
-          }
-        }
-      }
-      if (show) {
-        if (logger?.level === 'debug') {
-          msg += `${request.method} called with params:\n${inspectParams(request.params)}`
-        } else {
-          msg += `${request.method} called with params: ${inspectParams(request.params, 125)}`
-        }
-        logger?.info(msg)
-      }
+    if (checkFilter(request.method, rpcDebugVerbose)) {
```

Comment:
To note, the options seem to be:
* both `rpcDebugVerbose` and `rpcDebug` are true, in which case the full version will be logged
* both `rpcDebugVerbose` and `rpcDebug` are false, in which case nothing will be logged on requests and responses
* `rpcDebugVerbose` is true and `rpcDebug` is false, in which case the full version will be logged
* `rpcDebugVerbose` is false and `rpcDebug` is true, in which case the truncated version will be logged

Reply:
Yes, so for extra for clarification: the `true` and `false` are the returned values from this method, and are thus applied on the filter. If there is no filter (the filter is the empty string by default, so "no filter") then it always returns `false`.